packaging/Dockerfile

```diff
@@ -1,16 +1,18 @@
 ARG BASE_IMAGE=default
-FROM registry.opensource.zalan.do/library/alpine-3:latest AS default
+FROM alpine:latest AS default
```

Comment:
Why this change?

Reply:
I thought it would be more consistent:
- we override multiarch base image via build arg
- this Docker file by itself now depend on the stock alpine
- otherwise we would need to override base image for arm64 and armv7 (non-multiarch)

Reply:
If you switch the base image from a single-arch to a multi-arch it should just work (if the requested arch is supported, but alpine supports all we need). So you should be able to use `alpine:latest` for "arm64 and armv7 (non-multiarch)" as well.